plugins/config/functions

```diff
@@ -214,6 +214,7 @@ config_unset() {
     ENV_TEMP=$(echo -e "${ENV_TEMP}" | sed "/^export $var=/ d")
 
     config_write "$ENV_TEMP"
+    [[ "$var" == "NO_VHOST" ]] && config_set --no-restart $APP NO_VHOST=0 && DOKKU_CONFIG_RESTART=true
```

Comment:
it was impossible to support transitions from `NO_VHOST=1` to nothing and maintain a single function that told the truth about an app's vhost state.